location/src/main/resources/db/changelog/ddl/changelog-0004.sql

```diff
@@ -0,0 +1,10 @@
+--liquibase formatted sql
+
+--changeset dieunguyen:issue-001
```

Comment:
duplicate changeset with changelog-003

Reply:
if it's different file then the changeset id may be the same